### PR TITLE
Update payment - process integration

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-766

### Change description ###

integrate update payment process

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/PaymentMessageProcessorTest.java [hunk 1]
```diff
@@ -250,6 +345,52 @@ public void should_finalize_the_message_when_recoverable_failure_but_delivery_ma
         );
     }
 
+    @Test
+    public void should_finalize_update_message_when_recoverable_failure_but_delivery_maxed() throws Exception {
```
[the remark this answers is not in the document]
Comment (mdayican) [reply src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/PaymentMessageProcessorTest.java]: to be consistent  finalize?

[the remark this answers is not in the document]
Comment (mdayican) [reply src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/PaymentMessageProcessorTest.java]: changed all of them to be consistent :)

**Comment by apereverzin** [comment]
Please, have a look at this branch https://github.com/hmcts/bulk-scan-payment-processor/tree/new_update_process_integration, it suggests some refactoring.

**Comment by mdayican** [comment]
> Please, have a look at this branch https://github.com/hmcts/bulk-scan-payment-processor/tree/new_update_process_integration, it suggests some refactoring.

#72 has similar changes which is not liked. For now team prefer #82 for simplicity. in your branch I do not like the idea to create a different service for different messages. we can refactor the parser.

**Pull request merged**